form.yml.erb

```
@@ -22,10 +22,14 @@ form:
   - classroom
 attributes:
   jupyterlab_switch:
```

```suggestion
  mode:
```

Maybe if it had a better name it'd be more meaningful? How about `mode`? You're booting jupyter in a mode - lab or notebook.

If it had a meaningful name, it'd have a meaningful label.